tests/containers/buildah_image.pm

```diff
@@ -37,12 +37,15 @@ sub run {
         if (check_os_release('suse', 'PRETTY_NAME')) {
             # sle15-working-container is the default name given to a container. it is created in test_container_image
             test_opensuse_based_image(image => 'sle15-working-container', runtime => 'buildah');
-            allow_selected_insecure_registries(runtime => 'podman');
+            # Due to the steps from the test_opensuse_based_image previously,
+            # the image has been committed as refreshed
+            test_containered_app(runtime => 'buildah',
+                dockerfile => 'Dockerfile.sle',
+                base       => 'refreshed');
```

Comment:
interesting catch. i believe it is easy fix also. i will check

Reply:
after some self-review i have to say that the `refreshed` wouldnt break likely the opensuse jobs. the scope is not TW anyway at the current moment at the current task. the scheduling isnt used on o3. Although if it would, the static string used here (`sle15-working-container`) should change. i will do that in another PR, as this is out of this one